mantidproject/mantid
# Indirect - IqtFitMultiple and IqtFitSequential - Ability to extract fit members

Currently, the IqtFitMultiple and IqtFitSequential do not provide the option to extract the members from the fit, in a similar fashion to that of ConvolutionFitSequential.

This PR adds an 'ExtractMembers' property to the IqtFitMultiple and IqtFitSequential algorithm. When this property is set to True, the algorithms will extract the fit members and group them into a WorkspaceGroup.

The IqtFit interface has also been updated to display the ExtractMembers option as a checkbox in the fit property browser (in the same manner as within the ConvFit tab).

**To test:**
1. Navigate to Interfaces > Indirect > Data Analysis > IqtFit
2. Provide the selected file to the Input field.
3. Set the 'Exponential' spinner to ```2```
4. Check the check-box labelled 'Extract Members'
5. Click 'Run'
6. Ensure a ```*_Members``` workspace group is produced, containing the Data, Calc, Diff and two Exponential fit members.


**Test Files**
[IQTFit.zip](https://github.com/mantidproject/mantid/files/1710718/IQTFit.zip)

Fixes #21723 . 

**Release Notes** 
Release notes have been updated to include this change.


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
